.github/workflows/release.yml

```diff
@@ -141,5 +179,5 @@ jobs:
         if: ${{ env.TAG == 'latest' }}
         run: |
           git checkout develop
-          git merge master
+          git merge origin/master
```

Comment:
I don't know why, but `origin/` was required in my testing.

Reply:
Did master and origin/master match in your test environment?

Reply:
IIRC, locally, both work. When tested in GitHub Actions  origin/master (technically we tested the branch named origin/fix-ci) worked.

Reply:
I'm thinking it has something to do with the way it's checked out (it's not cloned!): https://github.com/trufflesuite/ganache/runs/6165271216?check_suite_focus=true#step:2:44

```
/usr/bin/git -c protocol.version=2 fetch --prune --progress --no-recurse-submodules origin +refs/heads/*:refs/remotes/origin/* +refs/tags/*:refs/tags/*
```